### DISCUSSION TITLE
Is it possible to print the first layers with 0.01 layer height and the next few layers with 0.05 layer height?

### DISCUSSION THREAD

**Discussion by denziri** [sn4k3/UVtools#921] [Q&A]
I want to combine 2 materials during printing, 1 material is very soft and has to be printed at 0.05 layer height, however I do not really understand the "dynamic layer height" setting. The idea is to switch to 0.05 layer height at a certain height.

**Comment by sn4k3** [comment]
First note that only a few printers can do what you want.

Dynamic layer height analyses layers and try to stack them to optimize the layer count, there is a quality threshold which tell how much to stack or skip that layer and leave as original.
This mean you have to slice your file at 0.01mm layers and let UVtools optimize up to 0.1mm or your max limit (can be 0.05mm).
This tool will kinda do what you want, however it may skip the stacking in favor of a quality requirement and then you can have from 0.01mm to 0.05mm layers. Also note you must set exposure time for each different heights as they require different time to optimal cure.

So this tool can do what you want at a lucky shot, other than this is possible only via a custom script.
Also note even with a costum script if the remaining layers are not multiple of 5 you have to decide either to discard the leftover or print them as different height.
